episodes/filters-and-thresholding.md

```diff
@@ -0,0 +1,743 @@
+for now.
+
+If you click on this option, you should see a new panel appear on the right side 
+of Napari:
+
+![](fig/gaussian-options.png){alt="Screenshot of settings for gaussian blur in 
+Napari" width='40%'}
+
+Make sure you have 'nuclei(data)' selected on the 'image' row, then click run:
+
+![](fig/nuclei-blurred-1.png){alt="Nuclei image after gaussian blur with sigma 
+of 1" width='40%'}
+
+You should see a new image appear in the layer list called 'Result of 
+gaussian_blur', which is a slightly blurred version of the original nuclei image.
+
+Try increasing the 'sigma' value to three and clicking run again:
+
+![](fig/nuclei-blurred-3.png){alt="Nuclei image after gaussian blur with sigma 
+of 3" width='40%'}
+
+You should see the nuclei become even more blurred in the new image.
+
+What's happening here? What exactly does a gaussian blur do? A gaussian blur is 
+an example of a 'linear filter' which is used to manipulate pixel values in 
+images. When a filter is applied to an image, each pixel value is replaced by 
+some combination of the pixel values around it. For example, below is shown a 
+small, zoomed-in area of the nucleus image. Here, the value of the red pixel 
+could be affected by all values within the displayed 3x3 box:
+
+![](fig/nuclei-kernel-area.png){alt="Small zoomed-in area of the nucleus image 
+with a pixel highlighted in red. Around this pixel is shown a 
+3x3 box." width="60%"}
+
+Exactly what this effect is, and the size of the region it involves, is 
+controlled by the filter's 'kernel'. The kernel can be thought of as another 
+very small image, where the pixel values control the effect of the filter. An 
+example 3x3 kernel is shown below:
+
+![](fig/nuclei-kernel.png){alt="Left - small area of the nucleus image with a 
+pixel highlighted in red. Around this pixel is shown a 3x3 box. Right - example 
+of a 3x3 kernel"}
+
+Bear in mind that kernels can have many different sizes! For example 3x3, 5x5, 
+9x9... To determine the new value of a pixel (like the red pixel above), the 
+kernel is placed over the target image centred on the pixel of interest. Then, 
+for each position in the kernel, we multiply the image pixel value with the 
+kernel pixel value. For example, multiplying the top left image pixel value with 
+the top left kernel value, then the top middle pixel value with the top middle 
+kernel value and so on... The values at each position are then summed together 
+to give the new value of the central pixel. To process an entire image, the 
+kernel is moved across the image pixel by pixel calculating the new value at 
+each location. Pete Bankhead's bioimage book has a 
+[really nice animation of this process
+](https://bioimagebook.github.io/chapters/2-processing/4-filters/filters.html#mean-filters)
+, which is worth taking a look at.
+
+For the gaussian filter we looked at above, the values in the kernel are based 
+on a 'Gaussian function'. A gaussian function creates a central peak that 
```

Comment:
Inconsistent capitalisation here. I'd go for Gaussian throughout but the main thing is it should be consistent.

Reply:
I went for 'gaussian' for now, as I mention it in at least one other episode as 'gaussian'. If we want to change to 'Gaussian' then probably best to open a separate issue and fix all episodes at once.